src/Microsoft.AspNet.Server.Kestrel/Http/MessageBody.cs

```diff
@@ -98,7 +99,6 @@ public static bool TryGet(IDictionary<string, StringValues> headers, string name
             return true;
         }
 
-
         class ForRemainingData : MessageBody
```

Comment:
`ForContentLength` with length 0. You can see how it's selected in [MassageBody.For](https://github.com/aspnet/KestrelHttpServer/blob/shalter/copy-to-less/src/Microsoft.AspNet.Server.Kestrel/Http/MessageBody.cs#L73).